packages/zoe/test/unitTests/contracts/test-atomicSwap.js

```diff
@@ -268,3 +268,122 @@ test('zoe - non-fungible atomicSwap', async t => {
   t.deepEquals(bobCcPurse.getCurrentAmount().extent, ['calico #37']);
   t.deepEquals(bobRpgPurse.getCurrentAmount().extent, []);
 });
+
+// Checking handling of duplicate issuers. I'd have preferred a raffle contract
+test('zoe - atomicSwap like-for-like', async t => {
+  t.plan(13);
+  const { moolaIssuer, moolaMint, moola } = setup();
+  const zoe = makeZoe({ require });
+  const inviteIssuer = zoe.getInviteIssuer();
+
+  // pack the contract
+  const { source, moduleFormat } = await bundleSource(atomicSwapRoot);
```

Comment:
for new-SES, follow the pattern from the other tests: use `bundle` instead of `{source, moduleFormat}`